R/lump.R

```diff
@@ -72,6 +72,9 @@ fct_lump <- function(f, n, prop, w = NULL, other_level = "Other",
   if (!xor(missing(n), missing(prop))) {
     new_levels <- ifelse(!in_smallest(count), levels, other_level)
   } else if (!missing(n)) {
+    if (!is.numeric(n)) {
+      stop("`n` must be numeric")
```

Comment:
Can you please use `rlang::abort()` here? (and below)

Reply:
done